blazor/datagrid/pdf-export.md

```diff
@@ -1035,6 +1035,60 @@ PDF export provides an option to export template columns of the DataGrid by defi
 }
 ```
 
+### Export filtered data only
+
+PDF export provides an option to export the filtered records by passing them to the [DataSource](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.PdfExportProperties.html#Syncfusion_Blazor_Grids_PdfExportProperties_DataSource) property of the [PdfExportProperties](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.PdfExportProperties.html) class in the [ToolbarClick](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridEvents-1.html#Syncfusion_Blazor_Grids_GridEvents_1_OnToolbarClick) event.
```

Comment:
by default filtered data only will be exported when calling the ExportToPdfAsync method. kindly share the purpose of this document

Reply:
i m porting this topic from EJ2. But in blazor by default filtered records are exported. so this topic is not valid to document.